### ISSUE TITLE
[RN 0.71.0-rc.5] newArchEnabled Project with path ':ReactAndroid' could not be found in project

### ISSUE THREAD

**Issue by leotm** [th3rdwave/react-native-safe-area-context#338]
`react-native-safe-area-context` builds fine w `newArchEnabled=false` but fails w `newArchEnabled=true`

```console
FAILURE: Build failed with an exception.

* Where:
Build file '/home/runner/work/RN071RC5/RN071RC5/node_modules/react-native-safe-area-context/android/build.gradle' line: 103

* What went wrong:
A problem occurred evaluating project ':react-native-safe-area-context'.
> Project with path ':ReactAndroid' could not be found in project ':react-native-safe-area-context'.
```

Repro/steps: https://github.com/leotm/RN071RC5/commits/master

nb: it's with few other libs too
- https://github.com/leotm/react-native-template-new-architecture/issues/1315

cc @tomekzaw + @software-mansion team looking at adding RN 0.71.0-rc.5 support too if helps
- https://github.com/software-mansion/react-native-reanimated/pull/3745
- https://github.com/software-mansion/react-native-gesture-handler/pull/2313 (in fabric example)
- https://github.com/software-mansion/react-native-screens/pull/1631 (in fabric example)

**Comment by klazbaba** [comment]
@leotm, were you able to resolve this. I am having the same issue.

**Comment by MateWW** [comment]
I've swapped that with the following and it went through.
Obviously, it needs a patch package
```
implementation("com.facebook.react:react-android")
```

**Comment by BubbleTrouble14** [comment]
@MateWW Could u maybe explain a little more in detail how you got it working, u added implementation("com.facebook.react:react-android") to the the react-native-safe-area-context gradle and then where did u add the second part ?

**Comment by cospin** [comment]
Hey @janicduplessis, could you please check this?
Now that RN 0.71 is out, and other major libs already support 0.71, react-native-safe-area-context is the only one missing to use other important packages like React Navigation.

**Comment by MateWW** [comment]
@BubbleTrouble14 
Sorry, I was writing it quickly without verifying.

To clarify I've posted 2 possible approaches:
1. Inside `react-native-safe-area-context` swap:
```
implementation project(":ReactAndroid")
```
```gradle
implementation("com.facebook.react:react-android")
```
This approach requires you to use patchPackage and may require the same for other libraries

2. Resolve all places where libraries are calling `:ReactAndroid`. 
What is probably a better solution till all libraries will resolve this issue
I've prepared an example working repository:
https://github.com/MateWW/react-native-71-missing-ReactAndroid/commit/bfbf73868ed54f8e999a05fbd7f8090de038e1f7
